### ISSUE TITLE
can u gime any example how to connect flash loan contract

### ISSUE THREAD

**Issue by gsngsn123** [Oluwatob-i/Arbitrage-Alphas#1]
hi greate work but  can u gime any example how to connect flash loan contract

**Comment by Oluwatob-i** [comment]
You can check this out https://docs.aave.com/developers/v/2.0/guides/flash-loans

**Comment by gsngsn123** [comment]
yeah but iam asking how to connect to this script can u pls help me pls
